package.json

```diff
@@ -39,6 +39,7 @@
     "typechain": "^8.1.1"
   },
   "dependencies": {
+    "@commander-js/extra-typings": "^11.0.0",
```

Comment:
Why are the typings needed as dependencies rather than dev only?
I guess this packaged doesn't contain only typings then

Reply:
My mistake, I initially started going down the bundler rabithole but quickly aborted. Forgot to revert this - have now done so.